ksql-engine/src/main/java/io/confluent/ksql/planner/plan/KsqlStructuredDataOutputNode.java

```diff
@@ -114,46 +114,36 @@ public KeyField getKeyField() {
     final QueryContext.Stacker contextStacker = builder.buildNodeContext(getId());
 
     final Set<Integer> rowkeyIndexes = SchemaUtil.getRowTimeRowKeyIndexes(getSchema());
-    final Builder outputNodeBuilder = new Builder(this);
     final Schema schema = SchemaUtil.removeImplicitRowTimeRowKeyFromSchema(getSchema());
-    outputNodeBuilder.withSchema(schema);
```

Comment:
This call could be replace with because later calls to `noRowKey.getSchema` with in this method can be inlined to `schema`.